apps/src/templates/teacherDashboard/EditSectionForm.jsx

```diff
@@ -218,6 +221,15 @@ class EditSectionForm extends Component {
             onChange={pairingAllowed => editSectionProperties({pairingAllowed})}
             disabled={isSaveInProgress}
           />
+          {ttsAvailable(section.scriptId) && (
```

Comment:
Passing a func to call here seems overly-complicated.  Would it be more straightforward just to get the list of sections that support TTS from redux or something like that?

Reply:
Sounds good, I'll update